frontend/src/components/accounts/AccountRow.tsx

```diff
@@ -34,6 +35,7 @@ export default class extends React.Component<Props, State> {
   render() {
     const { accountId, createdAt } = this.props;
     const { amount } = this.state;
+    let Id = accountId.length > 25 ? shortAccountId(accountId) : accountId;
```

Comment:
Let's move the check whether the accountId is too long inside the `shortAccountId` helper (let's have an optional second argument `lengthThreshold = 25`). I want to use it as:

```jsx
<Col className="transaction-row-title">{shortAccountId(accountId)}</Col>
```

P.S. `shortAccountId` is not a perfect naminng, so if you have a better option, please suggest